tests/framework/CodeInterfaceTests/CobraTF/CobraTFinterfaceTest4/testDummyStep_MC/2/vuq_param.txt

```diff
@@ -0,0 +1 @@
+k_db_1 = 1001.24174322
```

Comment:
Seems to me, you do not need this file. This file will generated by raven, right?